app/src/main/java/com/voipgrid/vialer/statistics/providers/DefaultDataProvider.java

```diff
@@ -97,6 +97,14 @@ public String getOSVersion() {
         return String.valueOf(Build.VERSION.SDK_INT);
     }
 
+    public String getDeviceManufacturer() {
+        return String.valueOf(Build.MANUFACTURER);
```

Comment:
String.valueOf is only needed when the parameter isn't already a string, both Build.Manufacturer/Model are already strings.